lib/manager/util/index.js

```diff
@@ -20,6 +18,8 @@ import type {
   ProjectsState
 } from '../../types/reducers'
 
+import {getVersionValidationSummaryByFilterStrategy} from './version'
+
```

Comment:
Undo this change.

Reply:
This change is made automatically during the eslint pre-commit hook. Is my eslint config broken/outdated? It's the one inside the mastarm folder inside `node_modules`